packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -772,9 +851,12 @@ export default class GoogleCalendarService implements Calendar {
     }
     const getCalIds = async () => {
       if (selectedCalendarIds.length !== 0) return selectedCalendarIds;
```

Comment:
if selectedCalendarIds is there use that, otherwise fallback to primary later in the code.